cortex_xdr_client/api/base_api.py

```diff
@@ -13,11 +13,11 @@ def __init__(self, auth: Authentication, fqdn: str, api_name: str, timeout: Tupl
         self._requests_timeout = timeout
         self._api_name = api_name
 
-    def _get_url(self, call_name: str) -> str:
+    def _get_url(self, call_name: Optional[str]) -> str:
```

Comment:
Why is this now optional?

Reply:
Forgot to change it back once I implemented a different solution for download_api.